### PR TITLE
Crosswalk

### PR DESCRIPTION
**Pull request by mphprogrammer**
Same actions as Weams in the previous PR. Basically cookie cutter.

### CODE REVIEW

**Comment by mphprogrammer** [comment]
@plusjeff I hope this isn't too big. Represents the addition of the VaCrosswalk resource. It is basically cookie cutter code with the Weams resource you looked at today. New Controller, View, Model, Migration, Specs. If its still too big, I can PR by actions...

**Comment by jeffmaher** [comment]
Thanks! Taking a look now.

**Comment by jeffmaher** [comment]
Can the .DS_Store files be added to .gitignore?

**Comment by mphprogrammer** [comment]
Thought I did ... Lemme look

Its in the gitignore, although maybe I have the notation wrong. ..

File: app/assets/javascripts/va_crosswalks.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (plusjeff) [inline app/assets/javascripts/va_crosswalks.js]: Can this scaffolding file get deleted?

File: app/assets/stylesheets/va_crosswalks.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the VaCrosswalks controller here.
```
Comment (plusjeff) [inline app/assets/stylesheets/va_crosswalks.scss]: Can this scaffolding file get deleted?

Comment (mphprogrammer) [inline app/assets/stylesheets/va_crosswalks.scss]: Yup - although I've been leaving them in to the end, on the off-chance I will need them at some point - but that's unlikely... I'll delete

**Comment by jeffmaher** [comment]
LGTM to me after the files are removed!

For .DS_Store, this might be helpful:
http://stackoverflow.com/a/107921/249016

**Comment by mphprogrammer** [comment]
Thanks Jeff, all the other PRs are pretty much the same

**Pull request merged**